src/agent/v8debugapi.ts

```diff
@@ -612,6 +612,8 @@ export function create(
       // TODO: Address the case where `breakpoint.expression` is `undefined`.
       const captured = state.capture(
           execState, breakpoint.expressions as string[], config, v8);
+      if (breakpoint.location && captured.location && captured.location.line)
```

Comment:
Can we move this +1 up to the place where `captured.location.line` is set?

Reply:
nit: use {} even for one-line inner statements, it helps with code readability.